File: ml-data.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
from ninstances_py.ninstances import ninstances
from lcard_py.lcard import lcard

def main():
    loop_condition=True

    while loop_condition == True:
        print("\nMenu:\n")
        print("0.- EXIT")
        print("1.- Number of instances, features and labels (Juan A. Romero)")
        print("2.- Label cardinality (Joaquín Quero Torres - i32qutoj)")

        main_input = int(input())

        if main_input == 0:
            loop_condition = False
            break
        elif main_input == 1:
            datafile=input("Introduce nombre de datafile: ")
            ninstances(datafile)
        elif main_input == 2:
            datafile=input("Introduce nombre de datafile: ")
            lcard(datafile)



if __name__ == "__main__":
    main()

File: ml-data.c
#include <stdio.h>
#include "ninstances-c/ninstances.h"
#include "statistics-c/statistics.h"

int main(void)
{
int x, flag;
flag=1;

while(flag==1)
{
printf("Menu:\n");
printf("0.- EXIT\n");
printf("1.- Number of instances, features and labels (Juan A. Romero)\n");
printf("2.- Statistics (Joaquín Quero Torres – i32qutoj)\n");
scanf("%d", &x);
fflush(stdin);
printf("\n\n");
switch(x){
    case 0:
        flag=0;
        break;
    case 1:
        printf("Ejecutando la función ninstances() de Juan A. Romero:\n");
        ninstances("yeast.complete");
        break;
    /* add here more functionality */
    case 2:
        printf("Ejecutando la función statistics() de Joaquín Quero Torres(i32qutoj):\n");
        statistics("scene.complete");
        break;
      }
}

}
